drivers/usb/usbstor_new/disk.c

```diff
@@ -170,12 +223,25 @@ USBSTOR_HandleInternalDeviceControl(
             Status = STATUS_SUCCESS;
             break;
         }
-
```

Comment:
Maybe keep a newline here to not clutter the different 'case's.

Reply:
Why divide all cases into two parts?

Reply:
It's just to have the different blocks visually separated.

Reply:
The brackets perform the separation in this case. As you can see, I only removed one space, not every case.
It was:
```
case
{
}
case
{
}
case
{
}

case
{
}
```
So it became:
```
case
{
}
case
{
}
case
{
}
case
{
}
```
Or do you suggest so?:
```
case
{
}

case
{
}

case
{
}

case
{
}

```